### DISCUSSION TITLE
Bytearrays for different Settings

### DISCUSSION THREAD

**Discussion by shubham9436** [gopro/OpenGoPro#110] [Q&A] [answered]
Hi,

Could you please enlist all the bytearray for each settings?
For example, for turning on Wifi, we use bytearray(b'\x03\x17\x01\x01') .

Though I need specifically for:

1. Turning On GPS
2. Auto Off
3. Photo Lens
4. Photo Timer
5. Photo Zoom
6. Shutter
7. White Balance
8. ISO Min, ISO Max
9. Sharpness
10. Color

**Comment by tcamise-gpsw** [comment, marked as answer]
Moving to discusion.

Hello. This information (at least for the supported features of the options you listed above) should be accessible from the BLE spec. For example, here is the [Command Quick Reference.](https://gopro.github.io/OpenGoPro/ble_2_0#commands-quick-reference)

Note that the following features you listed are not currently supported by the official Open GoPro API:
- turning on / off GPS
- Photo Timer
- white balance
- iso
- sharpness
- color

**Reply by hadoanngoc** [reply]
Hello, 

Quick app can control these settings. So I can assume that it is using unofficial API to connect/control GoPro?

**Reply by KonradIT** [reply]
You can just use the values from https://github.com/KonradIT/goprowifihack/blob/master/HERO11/HERO11-Commands.md as OpenGoPro settings IDs
